### PR TITLE
Support hierarchy change operations #8

### PR DESCRIPTION
**Pull request by soimugeoWB**
first we need to merge https://github.com/protegeproject/webprotege-backend-api/pull/8

### CODE REVIEW

File: src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java:85]: Replace `instanceof` checks with `.isNamed()`

File: src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java:91]: Use var keyword (same for all files)

File: src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java:111]: Replace this with a filter chain.  Replace `instanceof` with `.isNamed()`

File: src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/bulkop/EditParentsChangeListGenerator.java:122]: Rename to `addAddSubClassOfAxiomToChangeList`

File: src/main/java/edu/stanford/protege/webprotege/ProjectActionHandlerBeansConfiguration.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/ProjectActionHandlerBeansConfiguration.java:1]: Can you replace multiple imports with wild card imports?

File: src/main/java/edu/stanford/protege/webprotege/hierarchy/ClassHierarchyProviderImpl.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/hierarchy/ClassHierarchyProviderImpl.java:1]: Can you write a test for the cycle detection?  To make this easier it might be best to extract the cycle detection as a separate class that has a dependency to the hierarchy provide.  Your call.

File: src/main/java/edu/stanford/protege/webprotege/hierarchy/HierarchyCycleException.java
Comment (matthewhorridge) [inline src/main/java/edu/stanford/protege/webprotege/hierarchy/HierarchyCycleException.java:6]: Remove exclamation mark please

File: src/test/java/edu/stanford/protege/webprotege/dispatch/impl/ApplicationActionHandlerRegistry_Test.java
Comment (matthewhorridge) [inline src/test/java/edu/stanford/protege/webprotege/dispatch/impl/ApplicationActionHandlerRegistry_Test.java:31]: Do you need to remove the `webprotege.rabbitmq.commands-subscribe=false` here?

**Review by matthewhorridge — CHANGES_REQUESTED** [review]

**Comment by soimugeoWB** [comment]
Hello! i think it's done now. Thank you for the review!

File: src/main/resources/application-local.properties
Comment (alexsilaghi) [inline src/main/resources/application-local.properties:27]: please, dont leave it at debug.

**Review by alexsilaghi — COMMENTED** [review]
waiting for @matthewhorridge  decision with the events in FE

**Pull request merged**
